### PR TITLE
feat: first pass annotation api to fetch and get annotation

### PR DESCRIPTION
**Pull request by mwdchang**
### Summary
This add an Annotation table into terarium-postgres for tracking user-made annotations against some artifact. Also a couple of basic CRUD end points to.
- add annotation
- get annotation for (artifact_type, artifact_id)

#### Create annotation
`POST http://localhost:3000/api/annotations` a new annotation 
```
{
  "content": "Hello world",
  "artifact_type": "DEF",
  "artifact_id": "123"
}
```

#### Fetch annotations
`GET http://localhost:3000/api/annotations?annotation_id=123&annotation_type=DEF`

### CODE REVIEW

File: packages/services/hmi-server/src/main/java/software/uncharted/terarium/hmiserver/entities/Annotation.java [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+import javax.persistence.Index;
+import javax.persistence.Table;
+import java.io.Serializable;
+import java.time.Instant;
+import java.util.List;
+import java.util.UUID;
+
+
+/**
+ * Definition for use rannotations. An annotation can be applied to TERARium artiacts (e.g. document, model, dataset ...) as additional
```
Comment (dvince2) [inline packages/services/hmi-server/src/main/java/software/uncharted/terarium/hmiserver/entities/Annotation.java:24]: sp

File: packages/services/hmi-server/src/main/java/software/uncharted/terarium/hmiserver/resources/AnnotationResource.java [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+public class AnnotationResource {
+	@Inject
+	SecurityIdentity securityIdentity;
+
+	@GET
+	public Response getAnnotations(
+			@QueryParam("artifact_type") final String artifactType,
+			@QueryParam("artifact_id") final String artifactId) {
+
+			System.out.println("");
```
Comment (dvince2) [inline packages/services/hmi-server/src/main/java/software/uncharted/terarium/hmiserver/resources/AnnotationResource.java:31]: Probably want to remove debug statements here, or at least, replace with `log` if this is wanted output.

**Review by dvince2 — CHANGES_REQUESTED** [review]
LGTM - just the debug statements!

File: packages/services/hmi-server/src/main/java/software/uncharted/terarium/hmiserver/entities/Annotation.java (hunk 1 above)
Comment (mwdchang) [reply packages/services/hmi-server/src/main/java/software/uncharted/terarium/hmiserver/entities/Annotation.java]: fixed

**Review by YohannParis — APPROVED** [review]

**Pull request merged**